projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -8,6 +8,14 @@
 
 use Automattic\Jetpack\Connection\Client;
 
+if ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || Jetpack::is_connection_ready() ) {
+	// We load the necessary endpoints
+	require_once JETPACK__PLUGIN_DIR . '_inc/lib/class.memberships-rest-api-endpoints.php';
+
+	wpcom_rest_api_v2_load_plugin( 'WPCOM_REST_API_V2_Endpoint_Memberships' );
+	add_action( 'rest_api_init', array( 'Jetpack_Memberships_Json_Api_Endpoints', 'register_endpoints' ) );
```

Comment:
Here @lezama ( being a couple of flags). I haven't been able to test it, though

Reply:
for Jetpack only endpoints it seems it is better to register them here: https://github.com/Automattic/jetpack/pull/33786

if we do it here we would register the endpoints for simple sites too and I think it's not needed?